package.json

```diff
@@ -2,7 +2,59 @@
   "name": "monetr",
   "version": "0.1.0",
   "private": true,
-  "type": "module",
+  "scripts": {
+    "build": "rspack build",
+    "build:staging": "MONETR_ENV=staging rspack build --mode production",
+    "build:production": "yarn build --mode production",
+    "build-dev": "rspack --mode development",
+    "build-prod": "rspack --mode production",
+    "clean": "rm -rf build/* || true",
+    "start": "MONETR_ENV=${MONETR_ENV:=local} rspack serve --mode development",
+    "test": "jest",
+    "coverage": "jest --coverage --collectCoverageFrom=ui/**/*.{js,jsx,ts,tsx} --coverageDirectory=./reports",
```

Comment:
Since the collect coverage from is specified in the package.json for jest is the flag even necessary?